lighthouse-core/gather/gatherers/dobetterweb/domstats.js

```diff
@@ -77,20 +77,25 @@ function getDOMStats(element = document.body, deep = true) {
 /* c8 ignore stop */
 
 class DOMStats extends Gatherer {
+  /** @type {LH.Gatherer.GathererMeta} */
+  meta = {
+    supportedModes: ['snapshot', 'navigation'],
+  }
+
   /**
-   * @param {LH.Gatherer.PassContext} passContext
+   * @param {LH.Gatherer.FRTransitionalContext} passContext
    * @return {Promise<LH.Artifacts['DOMStats']>}
    */
-  async afterPass(passContext) {
+  async snapshot(passContext) {
     const driver = passContext.driver;
 
-    await driver.sendCommand('DOM.enable');
+    await driver.defaultSession.sendCommand('DOM.enable');
```

Comment:
hm, unrelated but that's weird why do we enable/disable DOM if everything is in `driver.evaluate` anyway? 🤔

Reply:
removed